test/integration/objectstore/_base.py

```diff
@@ -115,3 +128,120 @@ def setUp(self):
+        start_onezone(cls.oz_container_name)
+
+        oz_ip_address = get_onezone_ip_address(cls.oz_container_name)
+        start_oneprovider(cls.op_container_name, oz_ip_address)
+        await_oneprovider(cls.op_container_name)
+
+        super().setUpClass()
+
+    @classmethod
+    def tearDownClass(cls):
+        stop_docker(cls.op_container_name)
+        stop_docker(cls.oz_container_name)
+
+        super().tearDownClass()
+
+    @classmethod
+    def handle_galaxy_config_kwds(cls, config):
+        super().handle_galaxy_config_kwds(config)
+        temp_directory = cls._test_driver.mkdtemp()
+        cls.object_stores_parent = temp_directory
+        cls.object_store_cache_path = f"{temp_directory}/object_store_cache"
+        config_path = os.path.join(temp_directory, "object_store_conf.xml")
+        config["object_store_store_by"] = "uuid"
+        config["metadata_strategy"] = "extended"
+        config["outputs_to_working_directory"] = True
+        config["retry_metadata_internally"] = False
+        with open(config_path, "w") as f:
+            f.write(ONEDATA_OBJECT_STORE_CONFIG.safe_substitute(
+                {
+                    "temp_directory": temp_directory,
+                    "access_token": get_onedata_access_token(cls.oz_container_name),
+                    "onezone_domain": get_onezone_ip_address(cls.oz_container_name),
+                    "space_name": ONEDATA_DEMO_SPACE_NAME,
+                    "cache_updated_data": cls.updateCacheData(),
+                }
+            ))
+
+        config["object_store_config_file"] = config_path
+
+    def setUp(self):
+        super().setUp()
+        self.dataset_populator = DatasetPopulator(self.galaxy_interactor)
+
+    @classmethod
+    def updateCacheData(cls):
+        return True
+
+
+def start_onezone(oz_container_name):
+    cmd = [
+        "docker",
+        "run",
+        "-d",
+        "--name",
+        oz_container_name,
+        "--rm",
+        "docker.onedata.org/onezone:VFS-11786-fix-race-conditions-in-the-demo-dockers-curl-wrapper",  # TODO docker
+        "demo"
+    ]
+    subprocess.check_call(cmd)
+
+
+def get_onezone_ip_address(oz_container_name):
+    cmd = [
+        "docker",
+        "inspect",
+        '-f',
+        '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}',
+        oz_container_name,
+    ]
+    return subprocess.check_output(cmd).decode('utf-8').strip()
+
+
+def start_oneprovider(op_container_name, oz_ip_address):
+    cmd = [
+        "docker",
+        "run",
+        "-d",
+        "--name",
+        op_container_name,
+        "--rm",
+        "docker.onedata.org/oneprovider:VFS-11786-fix-race-conditions-in-the-demo-dockers-curl-wrapper",  # TODO docker
+        "demo",
+        oz_ip_address
+    ]
+    subprocess.check_call(cmd)
+
+
+def await_oneprovider(op_container_name):
+    cmd = [
+        "docker",
+        "exec",
+        op_container_name,
+        "await-demo"
+    ]
+    subprocess.check_call(cmd)
```

Comment:
ja to bym to wszystko robił inline o tak:
```
    subprocess.check_call([
        "docker",
        "exec",
        op_container_name,
        "await-demo"
    ])
```
imo czytelniej, ale co kto woli, więc możesz nie wprowadzać takich zmian